rmc/resources/reference_data.py

```diff
@@ -5,13 +5,20 @@
     VersionedTableResource,
 )
 
-from rmc.resources.basics import AMINO_ACIDS_PREFIX, RESOURCE_BUILD_PREFIX
-from rmc.resources.resource_utils import CURRENT_BUILD
+from rmc.resources.basics import AMINO_ACIDS_PREFIX, get_resource_build_prefix
+from rmc.resources.resource_utils import BUILDS, CURRENT_BUILD
 
-REF_DATA_PREFIX = f"{RESOURCE_BUILD_PREFIX}/reference_data"
-"""
-Path to bucket containing reference data resources.
-"""
+
+def get_ref_data_prefix(build: str = CURRENT_BUILD) -> str:
+    """
+    Get the path to the bucket for reference data resources.
+
+    :param build: Reference genome build. Default is `CURRENT_BUILD`.
+    :return: Path to bucket for reference data resources.
+    """
+    if build not in BUILDS:
+        raise DataException(f"Invalid build: {build}. Must be one of {BUILDS}!")
```

Comment:
Redundant with `get_resource_build_prefix` but not a biggie

Reply:
oh yeah I copied and pasted, but you're right it'll get checked there